app/services/projects_for_export_service.rb

```diff
@@ -1,14 +1,4 @@
 class ProjectsForExportService
-  def risk_protection_arrangement_projects(month:, year:)
```

Comment:
This came up when I was triaging - I decided I like the verbosity and link to what each export is called by users and told Sonarcloud to ignore this one, for me I'll take the duplication for the more 'obvioius' code - plus machines are silly - happy to keep this, or drop it and tell SC to ignore it as a personal preference

Reply:
I'm actually going to be removing those methods in a later PR (as we're getting rid of the RPA and funding agreement letters exports) so I'll leave these for now. But thanks for the heads up 👍